static/js/components/PersonalTab.js

```diff
@@ -25,19 +26,27 @@ class PersonalTab extends React.Component {
   updateGender(gender) {
     const { dispatch, profile: { profileCopy } } = this.props;
```

Comment:
yeah, I'd support a name change.

`profileCopy` is a little unclear - took me a bit to figure out why we were passing a copy of the whole thing back into the store, instead of updating the profile itself in the store.

maybe `unsavedProfile`, `profileEditInProgress`, something like that?